variables.tf

```diff
@@ -1,17 +1,23 @@
 variable "authorized_networks" {
   type = list(object({
-    name    = string
-    network = string
+    name            = string
+    network         = string
+    expiration_time = optional(string)
   }))
   default = []
 }
 
 variable "backup_config" {
   type = object({
-    enabled  = optional(bool)
-    location = optional(string)
+    enabled                        = optional(bool)
+    start_time                     = optional(string)
+    point_in_time_recovery_enabled = optional(bool)
+    location                       = optional(string)
+    transaction_log_retention_days = optional(number)
+    retained_backups               = optional(number)
```

Comment:
For most of these I couldn't determine defaults, hence making them optional.

Reply:
I'd think the "defaults" would be present in the state of all our existing instances? :thinking:

Reply:
Not in the terraform state apparently. Got them by fetching the data from api using `gcloud`.